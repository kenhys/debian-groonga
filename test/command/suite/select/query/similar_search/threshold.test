table_create Diaries TABLE_NO_KEY
column_create Diaries content COLUMN_SCALAR ShortText

table_create Terms TABLE_PAT_KEY ShortText \
  --default_tokenizer TokenBigram \
  --normalizer NormalizerAuto
column_create Terms diaries_content COLUMN_INDEX|WITH_POSITION Diaries content

load --table Diaries
[
{"content": "a a a b b c d e f g h i j k"}
]

select \
  --table Diaries \
  --match_columns content \
  --query '*S2"a b c"' \
  --output_columns content,_score \
  --sortby _id
