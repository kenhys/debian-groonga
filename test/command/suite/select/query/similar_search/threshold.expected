table_create Diaries TABLE_NO_KEY
[[0,0.0,0.0],true]
column_create Diaries content COLUMN_SCALAR ShortText
[[0,0.0,0.0],true]
table_create Terms TABLE_PAT_KEY ShortText   --default_tokenizer TokenBigram   --normalizer NormalizerAuto
[[0,0.0,0.0],true]
column_create Terms diaries_content COLUMN_INDEX|WITH_POSITION Diaries content
[[0,0.0,0.0],true]
load --table Diaries
[
{"content": "a a a b b c d e f g h i j k"}
]
[[0,0.0,0.0],1]
select   --table Diaries   --match_columns content   --query '*S2"a b c"'   --output_columns content,_score   --sortby _id
[
  [
    0,
    0.0,
    0.0
  ],
  [
    [
      [
        1
      ],
      [
        [
          "content",
          "ShortText"
        ],
        [
          "_score",
          "Int32"
        ]
      ],
      [
        "a a a b b c d e f g h i j k",
        2097155
      ]
    ]
  ]
]
